### ISSUE TITLE
v0.13.0 Beta5 - transparency on OSX - win title/name still shows with frames false.

### ISSUE THREAD

**Issue by Blatman** [nwjs/nw.js#4339]
Set the window.title in package.json to nothing and name is then used and is visible so have to set name to nothing as well - this will kill the app name up on the top OSX bar. I expected if frame was set to false then the title of the window should also be non-visible. Works fine with 0.12.3
Attached a wee test project to demonstrate.
Using Beta 5 30/1/16 on OSX 10.9.5 Mavericks. (Just place NWjs bin into TransTest folder to run).
[TransTest.zip](https://github.com/nwjs/nw.js/files/110557/TransTest.zip)

**Comment by fritx** [comment]
Try this:

```js
if (IS_MAC) {
  document.title = 'a' // must a falsy value
  document.title = '　' // a CJK white char
}
```

Then it works. We've hacked this in our production for a few months!

**Comment by Blatman** [comment]
Yup, seems to work quite well as a hack.
I just set win.title for OSX to the CKJ char (the magic bit) and the name in package.json is still usable.
I guess a fix would eventually be required to avoid the need for the hack of course.

Cheers.

**Comment by chino23** [comment]
could you reopen this so it can actually be fixed?

**Comment by chino23** [comment]
this fix doesn't seem to work for me

**Comment by Blatman** [comment]
Reopened - seems to need a proper fix.

**Comment by chino23** [comment]
thanks.

I think this issue started with 13.5. I didn't see this issue before, when using 13.4 and lower. But suddenly the window title appeared

**Comment by fritx** [comment]
The `document.title = '　'` hack works fine under nw 0.12.3

**Comment by chino23** [comment]
@rogerwang could we get a fix for this please as it's really ugly on OSX now with the title if we don't want it. Worked fine with beta4

**Comment by Blatman** [comment]
Closing this since was fixed in Git against #4507 and works OK in 0.13.0 final.
